packages/eas-cli/src/commands/device/delete.ts

```diff
@@ -0,0 +1,125 @@
+import { getConfig } from '@expo/config';
+import { Flags } from '@oclif/core';
+import assert from 'assert';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import {
+  chooseSingleDeviceToDeleteAsync,
+  formatDeviceLabel,
+} from '../../credentials/ios/actions/DeviceUtils';
+import { AppleDeviceMutation } from '../../credentials/ios/api/graphql/mutations/AppleDeviceMutation';
+import { AppleDeviceQuery } from '../../credentials/ios/api/graphql/queries/AppleDeviceQuery';
+import { AppleTeamQuery } from '../../credentials/ios/api/graphql/queries/AppleTeamQuery';
+import formatDevice from '../../devices/utils/formatDevice';
+import Log from '../../log';
+import { Ora, ora } from '../../ora';
+import { findProjectRootAsync, getProjectAccountNameAsync } from '../../project/projectUtils';
+import { promptAsync, toggleConfirmAsync } from '../../prompts';
+
+export default class DeviceDelete extends EasCommand {
+  static description = 'remove a registered device from your account';
+
+  static flags = {
+    'apple-team-id': Flags.string(),
```

Comment:
I'd be nice if this command accepted the device udid, so you could remove a device like this:
`eas device:delete --apple-team-id TEAM_ID --udid UDID`

Reply:
Parameter implemented, including multi-select